ToolBox/IBDownloader/Program.cs

```diff
@@ -0,0 +1,151 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using QuantConnect.Brokerages.InteractiveBrokers;
+using QuantConnect.Configuration;
+using QuantConnect.Data.Market;
+using QuantConnect.Logging;
+
+namespace QuantConnect.ToolBox.IBDownloader
+{
+    class Program
+    {
+        /// <summary>
+        /// Primary entry point to the program
+        /// </summary>
+        static void Main(string[] args)
+        {
+            if (args.Length != 5)
+            {
+                Console.WriteLine("Usage: IBDownloader SECURITY_TYPE SYMBOLS RESOLUTION FROMDATE TODATE");
+                Console.WriteLine("SECURITY_TYPE = eg Equity/Forex");
```

Comment:
No need to implement in this PR, is there a particular reason for not supporting Options and Futures as well?